From e909c988d0cfd88a2dd353a383693c11913f1a1f Mon Sep 17 00:00:00 2001
From: Marco Dalla Libera <marco.dallas95@gmail.com>
Date: Fri, 5 Jul 2013 17:52:49 +0200
Subject: [PATCH 6/6] Added call for AMD+ATI hardware configuration

01:05.0 VGA compatible controller [0300]: Advanced Micro Devices [AMD] nee ATI RS880M [Mobility Radeon HD 4225/4250] [1002:9712] (prog-if 00 [VGA controller])
02:00.0 VGA compatible controller [0300]: Advanced Micro Devices [AMD] nee ATI Madison [Mobility Radeon HD 5650/5750 / 6550M] [1002:68c1] (prog-if 00 [VGA controller])
---
 examples/turn_off_gpu.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/examples/turn_off_gpu.sh b/examples/turn_off_gpu.sh
index 37e26a0..a18daa7 100755
--- a/examples/turn_off_gpu.sh
+++ b/examples/turn_off_gpu.sh
@@ -30,6 +30,7 @@ methods="
 \_SB_.PCI0.PEGP.DGFX._OFF
 \_SB_.PCI0.VGA.PX02
 \_SB.PCI0.PEG0.PEGP.SGOF
+\_SB.PCI0.AGP.VGA.PX02
 "
 
 for m in $methods; do
-- 
1.8.2.1

